.github/workflows/romcal-ci.yml

```diff
@@ -23,10 +23,20 @@ jobs:
           node-version-file: '.nvmrc'
 
       # using -q to silent info outputs
-      - run: npm -q --no-progress ci --no-audit
+      - name: Install dependencies
+        run: npm -q --no-progress ci --no-audit
 
-      - run: npm run build
+      - name: Lint
+        run: npm run lint
 
-      - run: npm run test:without-coverage
+      - name: Check Formatting
```

Comment:
Inconsistent naming style: either change the name on L29 to `Check linting` (or something similar with `Check`) or naming this name (L32) to `Format`.

Reply:
the problem with a change like these is is that `Format` is connotative of "changing", whereas `Lint` is the generalized word for static analysis that doesn't change the code.

so: `Check Lint` is redundant, and `Format` is inaccurate, because we're doing read-only checking

Reply:
I agree that `Format` is inaccurate and `Check Format` is better.

> whereas `Lint` is the generalized word for static analysis that doesn't change the code.

While you are right by definition, however, ESLint _can_ and will change the code using `--fix`. But that is not the case in here.

---

I still prefer using `Check` for both of them, but that is my preference. We don’t like lint or format the code in the CI, thus naming them `Lint` or `Formatting` is inaccurate too. However, I don’t want to argue with you or talk much about this, as it is not an important thing. :wink: